scipy/stats/stats.py

```diff
@@ -5598,60 +5617,91 @@ def ttest_ind(a, b, axis=0, equal_var=True, nan_policy='propagate'):
 
     Notes
     -----
-    We can use this test, if we observe two independent samples from
-    the same or different population, e.g. exam scores of boys and
-    girls or of two ethnic groups. The test measures whether the
-    average (expected) value differs significantly across samples. If
-    we observe a large p-value, for example larger than 0.05 or 0.1,
-    then we cannot reject the null hypothesis of identical average scores.
-    If the p-value is smaller than the threshold, e.g. 1%, 5% or 10%,
-    then we reject the null hypothesis of equal averages.
+    Suppose we observe two independent samples, e.g. flower petal lengths, and
+    we are considering whether the two samples were drawn from the same
+    population (e.g. the same species of flower or two species with similar
+    petal characteristics) or two different populations.
+    The t-test quantifies the difference between the average (expected) values
+    of the samples, it and assesses the probability of such an observation
+    occuring by chance under the null hypothesis that the samples were drawn
+    from populations with the same average.
+    A p-value larger than our chosen threshold (e.g. 5% or 1%) indicates that
+    our observation is not so unlikely to have occured by chance, so we do
+    not reject the null hypothesis of identical averages.
```

Comment:
```suggestion
    not reject the null hypothesis of population means.
```

Reply:
Do you want it to just say "population means" or, e.g. "equal population means" like below?

Reply:
> Do you want it to just say "population means" or, e.g. "equal population means" like below?

equal, I edited just now to include, thanks.

Reply:
I will add this one manually to avoid triggering another build.